types/robust-segment-intersect/robust-segment-intersect-tests.ts

```diff
@@ -0,0 +1,10 @@
+import crosses from "robust-segment-intersect";
```

Comment:
Use the commonjs syntax `import crosses = require('robust-segment-intersect')`.

Consumers of the package who want to use `import crosses from ...` can add `"esModuleInterop": true`, but Definitely Typed tests need to be backward compatible to show that the types will work with old code bases.